roles/nova-data/tasks/libvirt.yml

```diff
@@ -95,3 +225,12 @@
 
 - name: add nova to the libvirtd group
   user: name=nova groups=libvirtd append=true system=yes createhome=no
+
+- name: ufw allow libvirtd ports for live migration
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 16509
+    - 16514
```

Comment:
Also, I think each node allows all incoming connections from every other node, https://github.com/blueboxgroup/ursula/blob/1d8027ad369bff96c935c3ad20ca6335c4b92e66/roles/common/tasks/ufw.yml#L15-L18  so this task can probably go away.